common/placeholders.js

```diff
@@ -56,9 +68,22 @@ function getPlaceholders(types) {
   if (types.find(type => type === placeholderTypes.ATTACHMENT))
     placeholders = placeholders.concat(getAttachmentPlaceholders())
 
+  if (types.find(type => type === placeholderTypes.TABLE))
+    placeholders = placeholders.concat(getTablePlaceholders())
+
   return placeholders
 }
 
+function table(value) {
+  let tableFinal = `<table><tr><td><p>NAME</p></td><td><p>VALUE</p></td></tr>`
```

Comment:
Not sure about all-caps NAME, VALUE and TOTAL. I'd use Item instead of NAME too.